src/mon/MonCommands.h

```diff
@@ -1030,7 +1030,8 @@ COMMAND("osd pool set "
 	"name=pool,type=CephPoolname "
 	"name=var,type=CephChoices,strings=size|min_size|pg_num|pgp_num|pgp_num_actual|crush_rule|hashpspool|nodelete|nopgchange|nosizechange|write_fadvise_dontneed|noscrub|nodeep-scrub|hit_set_type|hit_set_period|hit_set_count|hit_set_fpp|use_gmt_hitset|target_max_bytes|target_max_objects|cache_target_dirty_ratio|cache_target_dirty_high_ratio|cache_target_full_ratio|cache_min_flush_age|cache_min_evict_age|min_read_recency_for_promote|min_write_recency_for_promote|fast_read|hit_set_grade_decay_rate|hit_set_search_last_n|scrub_min_interval|scrub_max_interval|deep_scrub_interval|recovery_priority|recovery_op_priority|scrub_priority|compression_mode|compression_algorithm|compression_required_ratio|compression_max_blob_size|compression_min_blob_size|csum_type|csum_min_block|csum_max_block|allow_ec_overwrites|fingerprint_algorithm|pg_autoscale_mode|pg_autoscale_bias|pg_num_min|target_size_bytes|target_size_ratio "
 	"name=val,type=CephString "
-	"name=yes_i_really_mean_it,type=CephBool,req=false",
+	"name=yes_i_really_mean_it,type=CephBool,req=false "
+	"name=yes_i_really_really_mean_it,type=CephBool,req=false",
```

Comment:
IMHO, I don't really like the syntax. 
FWIW,  can't you simply reuse the existing  yes_i_really_mean_it to achieve the same purpose?

Reply:
@xiexingguo yeah, that would be good alternative, made necessary changes.